lib/questions.js

```diff
@@ -160,13 +170,11 @@ module.exports = {
         message: messages.breaking,
         when(answers) {
           // eslint-disable-next-line max-len
-          if (
+          return !!(
             config.askForBreakingChangeFirst ||
             (config.allowBreakingChanges && config.allowBreakingChanges.indexOf(answers.type.toLowerCase()) >= 0)
-          ) {
-            return true;
-          }
-          return false; // no breaking changes allowed unless specifed
+          );
+          // no breaking changes allowed unless specifed
```

Comment:
We may not need this comment anymore with the direct function return.

Reply:
Yes, you can remove comments